src/main/java/com/autotune/utils/KruizeConstants.java

```diff
@@ -405,6 +412,21 @@ public static final class DATABASE_ENV_NAME {
         public static final String DATABASE_SSL_MODE = "database_sslmode";
     }
 
+    /**
+     * In order to assign values to the static variables of KruizeDeploymentInfo
+     * using Java reflection, the class variables are utilized, and therefore,
+     * if any new variables are added, their corresponding declaration is necessary.
+     * Ref InitializeDeployment.setConfigValues(KruizeConstants.CONFIG_FILE, KruizeConstants.KRUIZE_CONFIG_ENV_NAME.class);
+     */
+    public static final class CLOUDWATCH_LOGS_ENV_NAME {
```

Comment:
should be moved inside KRUIZE_CONFIG_ENV_NAME 
and value should have only one underscore